### PR TITLE
[MDB IGNORE] [IDB IGNORE] Upstream merge 11-5 + some other changes

### PR DESCRIPTION
**Pull request by MrMelbert**
- Upstream merges to 11-5 + the locker based combat revert PR because it's broken
- Updated the merge driver.
- Redid all the accesses for modular jobs. Some minor changes were made here. 
- Got rid of a lot of un-needed modular changes. (By god, I want to revert the 50 year pr)
- Updated the sprite for the painkiller pill bottle. 
- Fixed Lima and Pubby, too. 

- ### Renamed Toxicologist to Ordnance Technician

### CODE REVIEW

**Comment by Jolly-66** [comment]
Permission to revert the stupid year PR 👍

**Comment by AffectedArc07** [comment]
Please also put `[IDB IGNORE]` in the title.

You crashed it.
image

**Comment by MrMelbert** [comment]
whoops

**Comment by Jolly-66** [comment]
> Please also put `[IDB IGNORE]` in the title.
> 
> You crashed it. image

Sorry for that!

File: jollystation_modules/code/modules/jobs/job_types/shaft_miner.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// -- Shaft miner changes --
+/datum/job/shaft_miner
+	department_head = list("Quartermaster") // NON-MODULE CHANGE
+	supervisors = "the quartermaster" // NON-MODULE CHANGE
```
Comment (Jolly-66) [inline jollystation_modules/code/modules/jobs/job_types/shaft_miner.dm:4]: comment

**Comment by MrMelbert** [comment]
I'm just going to merge this now since all checks pass, fix any fallout later, so I can work on other fixes following today's session

**Pull request merged**